### PR TITLE
Adding fake token warning and replacing `Add Token` labels with `Import Tokens`

### PR DESCRIPTION
**Pull request by NiranjanaBinoy**
This PR is a part of issue #11619 
This PR replaces the `Add Tokens` labels with `Import Tokens` and a warning message on the custom token addition page for scam and security risk awareness.

Screen Shot 2021-09-03 at 3 44 22 PM

Screen Shot 2021-09-03 at 3 44 46 PM

Screen Shot 2021-08-09 at 10 15 46 PM

Screen Shot 2021-08-09 at 10 17 30 PM

Screen Shot 2021-08-09 at 10 16 38 PM

### CODE REVIEW

File: ui/pages/import-token/import-token.component.js
Comment (ryanml) [inline ui/pages/import-token/import-token.component.js:278]: nit, fix leading whitespace in `href`

File: test/e2e/tests/permissions.spec.js
Comment (adonesky1) [inline test/e2e/tests/permissions.spec.js:65]: Hmmm why was this change required?

**Review by adonesky1 — COMMENTED** [review]
[nit] seems like you should rename the file and component in confirm-add-token.stories.js as well

File: test/e2e/tests/permissions.spec.js
Comment (NiranjanaBinoy) [reply test/e2e/tests/permissions.spec.js:65]: The test is breaking without that delay

**Review by adonesky1 — DISMISSED** [review]
LGTM!

File: ui/pages/import-token/import-token.stories.js
Comment (darkwing) [inline ui/pages/import-token/import-token.stories.js:11]: This needs to be `Boolean`

Comment (brad-decker) [reply ui/pages/import-token/import-token.stories.js:11]: When did you change your username, Mark? (cc @Gudahtt, lol)

Comment (NiranjanaBinoy) [reply ui/pages/import-token/import-token.stories.js:11]: This seems to be the `boolean` function imported from `@storybook/addon-knobs`.  I haven't made any change in the file other than renaming it.

Comment (brad-decker) [reply ui/pages/import-token/import-token.stories.js:11]: ah! oops.

**Review by brad-decker — DISMISSED** [review]

File: ui/pages/pages.scss
Comment (mcmire) [inline ui/pages/pages.scss:2]: Should this be moved down to make this list alphabetical as per the comment above?

**Review by danjm — DISMISSED** [review]
LGTM once merge conflicts are resolved

File: ui/pages/pages.scss
Comment (danjm) [reply ui/pages/pages.scss:2]: I'm okay with not enforcing import ordering that isn't yet enforced by our linter

Comment (brad-decker) [reply ui/pages/pages.scss:2]: Sounds like a challenge. 🥇

**Review by brad-decker — APPROVED** [review]

**Review by danjm — APPROVED** [review]

**Pull request merged**
